mkt/comm/models.py

```diff
@@ -72,17 +73,27 @@ def check_acls_comm_obj(obj, profile):
     return False
 
 
-def user_has_perm_app(user, app):
+def user_has_perm_app(user, obj):
     """
+    It's named `app` for historical reasons, but it `obj` can be either a
+    Webapp or Extension.
+
     Check if user has any app-level ACLs.
     (Mozilla contact, admin, review, senior reviewer, developer).
     """
+    # grep: comm-content-type.
+    has_perm = False
+    if obj.__class__ is Webapp:
```

Comment:
`is` works, but `==` is safer here as you are not comparing identity, you are comparing equality.